package.json

```diff
@@ -17,8 +17,9 @@
     "format:check": "yarn prettier-base --check",
     "cypress": "cypress open",
     "cypress:headless": "cypress run",
-    "e2e": "start-server-and-test 'yarn dev' http://localhost:3000 'yarn cypress'",
-    "e2e:headless": "start-server-and-test 'yarn serve' http://localhost:3000 'yarn cypress:headless'",
+    "e2e": "start-server-and-test 'yarn build && yarn serve' http://localhost:3000 'yarn cypress'",
```

Comment:
Next.js v12 with SWC now builds the production bundle so fast that I've switched the default `e2e` command to build and serve the prod bundle rather than running the local dev server (still available under `e2e:dev` tho). This way, when navigating pages the E2E tests don't need to wait for building the page on-fly (causing delays in assertions etc).

Reply:
Cool that means we could reduce the wait statements on Cypress as well, because I used that wait partially to account for the dev build when hitting the comparator page. We still need a reasonable wait for the GitHub API response though.